modules/logging-release-notes-5-8-0.adoc

```diff
@@ -0,0 +1,56 @@
+//module included in logging-5-8-release-notes.adoc
+:content-type: REFERENCE
+[id="logging-release-notes-5-8-0_{context}"]
+= Logging 5.8.0
+This release includes link:https://access.redhat.com/errata/RHBA-2023:6139[OpenShift Logging Bug Fix Release 5.8.0] and link:https://access.redhat.com/errata/RHBA-2023:6134[OpenShift Logging Bug Fix Release 5.8.0 Kibana].
+
```

Comment:
Please add the following: 
```

== Deprecation notice
In Logging 5.8, Elasticsearch, Fluentd, and Kibana are deprecated and are planned to be removed in Logging 6.0, which is expected to be shipped alongside {product-title} version 4.16. Red Hat will provide critical and above CVE bug fixes and support for these components during the current release lifecycle, but these components will no longer receive feature enhancements. The Vector-based collector provided by the Cluster Logging Operator, and LokiStack provided by the Loki Operator are the preferred operators for log collection and storage.  We encourage all users to adopt the Vector and Loki log stack, as this will be the stack that will be enhanced going forward.
```

Reply:
Done! Thanks Libby.

Reply:
@shreyasiddhartha - Thank you!  We may need to change the wording of this, need to check in with Jamie before a merge is requested to check.